doc/source/operations/upgrading-openstack.rst

```diff
@@ -54,6 +112,24 @@ using Heat, and disable the service.
 
 TODO: guide for disabling Heat
 
+Designate sink disabled by default
+----------------------------------
+
+Designate sink is optional designate service which listens for event
+Notifications, primarily from Nova and Neutron. It is disabled by default (when
+designate is enabled) in Caracal. It is not required for Designate to function.
```

Comment:
Designate sink is an optional Designate service which listens for event
notifications, primarily from Nova and Neutron. It is disabled by default (when
designate is enabled) in Caracal. It is not required for Designate to function.